strawman-Array.prototype.last.md

```diff
@@ -0,0 +1,62 @@
+# Array.prototype.last
+A proposal for an ECMAScript native method
+
+Code
+---
+
+    //everything in 6 lines
+    Array.prototype.last = function () {
```

Comment:
[/u/defproc](http://www.reddit.com/user/defproc) [proposed this](http://www.reddit.com/r/javascript/comments/363qly/arrayprototypelast/crar203) on Reddit.

```javascript
//everything in 6 lines
Array.prototype.last = function () {
    if (this.length === 0) {
        return undefined;
    }
    return this[this.length - 1];
};
```

**I take absolutely zero credit for this!** All credit goes to [/u/defproc](http://www.reddit.com/user/defproc) on Reddit. I just didn't want his suggestion to end being just a stale comment on the Internet; I wanted it to be an actual part of the discussion of this feature.